cmake/FindPostgreSQL.cmake

```diff
@@ -125,6 +125,9 @@ if(NOT DEFINED PG_CONFIG)
             set(OLD_LDFLAGS "$ENV{LDFLAGS}")
             set(ENV{CFLAGS} "${OLD_CFLAGS} -I ${OPENSSL_ROOT_DIR}/include")
             set(ENV{LDFLAGS} "${OLD_LDLAGS} -L ${OPENSSL_ROOT_DIR}/lib")
+            # This ensure whatever compiler we're using to compile Omnigres, we're going
+            # to sue the same for Postgres
```

Comment:
s/sue/use?

Reply:
yes, last working draft has this fixed already